js/hacks/hack1_NIKO_HACKED_YOU/hack1_story.js

```diff
@@ -334,17 +332,14 @@ monogatari.script ({
         "show character n slight_smile at right with fadeIn",
         "show character c neutral_closed at left with fadeIn",
 
-        "n Nice one Clara! That was a pretty tough problem.",
-        "n It took me way longer to finish when I did it.",
-        "n I have to say, the Hacker Boyz are definitely going to reconsider your application",
+        "n Nice one Clara! That was a pretty tough problem. It took me way longer to finish when I did it. The Hacker Boyz are definitely going to reconsider your application.",
```

Comment:
Is there a particular reason why we name it "Boyz." I am not sure about the naming because it sounds not really professional I feel like?

Reply:
It's the kind of name that teens might give their club?

Reply:
That makes sense!